### ISSUE TITLE
[A11Y] Login and registration forms missing instructions for required fields

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9370]
## Issue Description
All required form fields are visually marked with an asterisk (*), but there is no instruction indicating the required state when using this resource.

## How to reproduce / Current behavior
1. Access an OJS  installation 
2. Click on top right link "Login" or "Register"
3. See the form with asterisk (*) on required fields, but no instruction about the required state.

## What application are you using?
 OJS version 3.3.0.13

## Stack used 
- Device: Laptop
- OS: MacOS(Monterey)
- Browser: Firefox
- Version: 118.0.1 (64-bit)

## Testing method
- Method:  Manual

## Action
It should be include, above the form fields, a paragraph stating "Required fields  marked with *". E.g.:

`<p> Required fields are marked with an asterisk (<abbr class="req" title="required">*</abbr>).</p>`

## Additional information

Current interface:
![image](https://github.com/pkp/pkp-lib/assets/4992902/d618021e-9a11-49a0-8b2b-4ec26600ca34)

How it should be:
![image](https://github.com/pkp/pkp-lib/assets/4992902/6043b21a-fcff-47f1-ab60-4241979fe28f)
